src/Umbraco.Web.BackOffice/Controllers/DataTypeController.cs

```diff
@@ -348,6 +348,33 @@ public IActionResult PostMove(MoveOrCopy move)
             }
         }
 
+        public IActionResult PostCopy(MoveOrCopy copy)
+        {
+            var toCopy = _dataTypeService.GetDataType(copy.Id);
+            if (toCopy == null)
+            {
+                return NotFound();
+            }
```

Comment:
Being nitpicky again, but pattern matching is the new hotness 🔥 
```suggestion
            if (toCopy is not null)
            {
                return NotFound();
            }
```

Reply:
@Zeegaan shouldn't be `is null`?
I guess it shouldn't return not found if the entity isn't null.

Reply:
@bjarnef Good catch! You are totally right 👍

Reply:
Updated!